### DISCUSSION TITLE
修改breakpoint.less里面的css变量，在页面上并没有生效

### DISCUSSION THREAD

**Discussion by owl-lei** [vbenjs/vue-vben-admin#2963] [Q&A]
在Login.vue组件，使用了@screen-xl

    @media (max-width: @screen-xl) {
      background-color: #293146;

      .@{prefix-cls}-form {
        background-color: #fff;
      }
    }

页面渲染出来的是max-width:1200px;我想把他改为1280，跟unocss里的xl:text-gray-600尺寸保持一致，
但修改breakpoint.less里面的@screen-xl: 1280px;并没有生效。

我改如何做才能调整@screen-xl的值？

**Comment by likui628** [comment]
找到问题的原因了
https://github.com/vbenjs/vue-vben-admin/blob/e31525a803ee5a7c9770b4eccebe91cc90f7aa60/internal/vite-config/src/utils/modifyVars.ts#L28
你可以打印一下,把我们预设的值给覆盖了
```js
 console.log(modifyVars);  
/*
{
  ...
  'screen-xl': '1200px',
}
*/
```
